sites/default/menu_data.json

```diff
@@ -625,23 +625,23 @@
             "children" : [ ],
             "requirement" : 0
         }, { 
-            "label" : "Chart Activity",
+            "label" : "Chart Location Activity",
             "menu_id" : "rep0",
             "target" : "rep",
             "url" : "/interface/reports/chart_location_activity.php",
             "children" : [ ],
             "requirement" : 0,
             "global_req" : "!disable_chart_tracker"
         }, { 
-            "label" : "Charts Out",
+            "label" : "Charts Checked Out",
```

Comment:
Or this

Reply:
The near future will decide the fate of these so called reports.